### PR TITLE
adds category, product, review models

### PR DESCRIPTION
**Pull request by lcbuchanan**
### Assignee Tasks

- [ ] added unit tests (or none needed)
- [ ] written relevant docs (or none needed)
- [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

*Your PR Notes Here*

### CODE REVIEW

File: server/db/models/category.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Category = db.define('category', {
+  name: {
+    type: Sequelize.STRING,
+    len: [2, 15],
```
Comment (avbraun) [inline server/db/models/category.js]: This should probably move to 'validate'!

File: server/db/models/product.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    type: Sequelize.INTEGER,
+    allowNull: false
+  },
+  isAvailable: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: true
+  },
+  photoUrl: {
+    type: Sequelize.STRING,
+    defaultValue:' https://cdn.shopify.com/s/files/1/2607/2780/products/socks-template.jpg?v=1512403868'
```
Comment (avbraun) [inline server/db/models/product.js]: Extra space in between ' and https!

File: server/db/models/product.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  }
+})
+
+module.exports = Product;
+
+/**
+ * instanceMethods
+ */
+Product.prototype.priceDollars  = function () {
+  const dollarPrice = this.price / 200;
```
Comment (avbraun) [inline server/db/models/product.js]: Oops! 100. :)

File: server/db/models/category.js (hunk 1 above)
Comment (lcbuchanan) [reply server/db/models/category.js]: fixed!

File: server/db/models/product.js (hunk 2 above)
Comment (lcbuchanan) [reply server/db/models/product.js]: deleted the space!

File: server/db/models/product.js (hunk 3 above)
Comment (lcbuchanan) [reply server/db/models/product.js]: changed to 100

**Pull request merged**